src/gv-styles/_colors.scss

```diff
@@ -13,6 +13,7 @@ $switch-handler-color: #fff;
 $separator-color: #28323b;
 
 $background-color: #131e26;
+$secondary-background-color: #1f2b36;
```

Comment:
Please add this color to colors.ts
https://github.com/GenesisVision/gv-react-components/blob/master/src/gv-styles/colors.ts

Reply:
Done!